.releaserc

```diff
@@ -1,9 +1,8 @@
 {
   "branches": [
-    "master",
-    "next",
-    { "name": "alpha", "prerelease": true },
-    { "name": "beta", "prerelease": true }
+    { "name": "release-22.x", "range": "22.x", "channel": "22.x" },
+    { "name": "release-23.x", "range": "23.x" },
+    { "name": "next", "prerelease": true, "channel": "next" }
```

Comment:
I'm not quite sure how `prerelease` and `channel` interact, but of all things to "test in prod" about this config it feels the least scary.